.github/workflows/ci_integrations_actions.yml

```diff
@@ -91,10 +91,40 @@ jobs:
           name: ${{ matrix.output_name }}
           path: metrics/ci_integrations/${{ matrix.output_name }}
           if-no-files-found: error
+  
+  ci-integrations-linux-generate-sha256-actions:
+    name: CI Integrations Linux Generate SHA256 Actions
+    needs: [ ci-integrations-build-actions ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ref: ${{ github.ref }}
+      
+      - name: Download Linux Build
+        uses: actions/download-artifact@v2
+        with:
+          name: ci_integrations_linux
+          path: metrics/ci_integrations/
+
+      - name: Create CI Integrations Linux Archive
```

Comment:
It seems like we should generate a checksum for a file, but not an archive with this file. Please, apply this suggestion to the `Coverage Converter` checksum generation.